--- node_modules/dexie/dist/dexie.mjs	2024-10-25 10:28:58.955692160 +0200
+++ dexie.mjs	2024-10-25 10:28:31.467678581 +0200
@@ -5619,6 +5619,39 @@
         indexedDB: _global.indexedDB || _global.mozIndexedDB || _global.webkitIndexedDB || _global.msIndexedDB,
         IDBKeyRange: _global.IDBKeyRange || _global.webkitIDBKeyRange
     };
+
+    // this is a patch, which sequres the access to indexedDB, so that only
+    // LiaScript can access and store this data ...
+    try { 
+        delete _global.indexedDB; 
+    } catch(e) { 
+        _global["indexedDB"] = undefined; 
+    }
+    try { 
+        delete _global.mozIndexedDB; 
+    } catch(e) { 
+        _global["mozIndexedDB"] = undefined; 
+    }
+    try { 
+        delete _global.webkitIndexedDB; 
+    } catch(e) { 
+        _global["webkitIndexedDB"] = undefined; 
+    }
+    try { 
+        delete _global.msIndexedDB; 
+    } catch(e) { 
+        _global["msIndexedDB"] = undefined; 
+    }
+    try { 
+        delete _global.IDBKeyRange; 
+    } catch(e) { 
+        _global["IDBKeyRange"] = undefined; 
+    }
+    try { 
+        delete _global.webkitIDBKeyRange; 
+    } catch(e) { 
+        _global["webkitIDBKeyRange"] = undefined; 
+    }
 }
 catch (e) {
     domDeps = { indexedDB: null, IDBKeyRange: null };
